prclz/etl/split_bldgs.py

```diff
@@ -0,0 +1,104 @@
+import argparse
```

Comment:
we can kill most of these imports when we move the cli wiring to cli.py

Reply:
removed